libraries/rush-lib/src/logic/operations/test/__snapshots__/BuildPlanPlugin.test.ts.snap

```diff
@@ -0,0 +1,736 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BuildPlanPlugin build plan debugging should generate a build plan 1`] = `
+"Build Plan Depth (deepest dependency tree): 5
+Build Plan Width (maximum parallelism): 40
+Number of Nodes per Depth: 1, 4, 11, 40, 34
```

Comment:
What does this mean?

Reply:
At 
- depth 0, there's 1 node, 
- depth 1, 4 nodes
- depth 2, 11 nodes
and so on.

My goal was to show how spread out the tree is, the more nodes there are per depth the better your maximum parallelization is.